### DISCUSSION TITLE
Is it possible to get an invoice for EU company (reverse charge) for a personal Github account

### DISCUSSION THREAD

**Discussion by dcarapic** [community/community#48483] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I have a company in Croatia (EU) and I would like to purchase copilot on my company with reverse charge.
Is this possible?

**Comment by iamcjwilliams** [comment]
@dcarapic The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)

**Comment by katsar0v** [comment]
@dcarapic do you have an update on this issue? I experience a similar problem and only got a sample receipt with no address.

**Reply by dcarapic** [reply]
I got this reply:

> 
> Hi Dalibor,
> 
> Thanks for contacting us about this. GitHub's head offices and servers are located in the United States, but we are compliant with EU regulations like GDPR.
> 
> We do not provide invoices, but if you need specific information added to your receipts, such as company name, address or VAT/GST, you can add the information by going to your account settings page and clicking on the "Add information" button. This will bring up a free-form text box where you can enter whatever information you need. In case you need it, we have a guide that'll walk you through this:
> 
> [Adding information to your receipts](https://docs.github.com/billing/managing-your-github-billing-settings/adding-information-to-your-receipts)
> 
> Once you have added the information, you can download a past receipt with that newly-added info by following the instructions here:
> 
> [Viewing your payment history and receipts](https://docs.github.com/billing/managing-your-github-billing-settings/viewing-your-payment-history-and-receipts)
> 
> Please let us know if you have any further questions.
> 
> Best regards,
> Anna

I do not know if this helps you or not. It is still a receipt and not a real invoice. You can add a company name and that's about it.

**Reply by peturingi** [reply]
This discussion is the first result on google when searching for: github vat receipt eu

On the receipt you sent to my company:

Total: $10.00 USD*
_* VAT/GST paid directly by GitHub, where applicable_

Given that GitHub is registered as company with a VAT number in Denmark, (see https://datacvr.virk.dk/enhed/virksomhed/36486627)
it is unclear to me what "where applicable" means.

Is this a reverse charge?

**Comment by IgorStrapko** [comment]
I have the same question: I livein the UK, I need to register the GitHub Pilot transaction in my company's QuickBooks, should I mark it as Reverse Charged or No VAT at all?

**Comment by cannuri** [comment]
how did y'all solve this?

**Comment by eadrwlo** [comment]
I am also interested what happened?
 It seems Github is doing something wrong. I am paying also for ChatGPT and I am receiving real invoice with Open AI EU OSS VAT number and the price is lower by 23% tax because I am providing my VAT EU TAX ID.
 For Github it seems it should be the same, when we are buying it as a company (whatever Croatia, Poland etc) we should have lower price with reverse charge.

**Reply by jozefizso** [reply]
Yes.

**Comment by thurti** [comment]
For all coming from google. There is now a setting on the [Payment Information](https://github.com/settings/billing/payment_information) page to receive a proper invoice (scroll down). 

You can also download older invoices [Payment History](https://github.com/account/billing/history). The "Bill To" is still "undefined, undefined" in my case (*must resist to insert Copliot reference here*), but hey - at least an invoice number and a note on reverse charge.

**Reply by jozefizso** [reply]
Thanks for this info!

I tried to download the invoice - I had my company legal name, VAT ID and other info already setup in my GitHub account, but the invoice is garbage, the bill to field is nonsense:

<img width="231" alt="image" src="https://github.com/user-attachments/assets/4eab94fd-b46d-4cdd-ba3b-b316e6264220" />

**Comment by derFunk** [comment]
For the invoice to be "proper" it still should not be called "receipt" though...

Also the question about GitHub paying VAT is not resolved yet - if they forward the VAT we don't have to declare it as reverse charge. If not, we have to pay taxes. "*VAT is included in the sales price when and where applicable." is a pretty confusing sentence on the invoice.

Worst case is we pay double the taxes. Anyone with more infos here how GitHub follows tax laws?

My current take is that it's reverse charge, better safe than sorry.

**Comment by stefanoconiglio** [comment]
Dear all, any updates on the VAT reverse charge issue?

**Reply by cdorn** [reply]
there is now an option to go to payment history that offers an invoice (had my correct details and said to apply reverse charge)
